src/openassetio-python/_openassetio.hpp

```diff
@@ -11,6 +11,31 @@
 
 #include <openassetio/typedefs.hpp>
 
+#include "PyRetainingSharedPtr.hpp"
+
+OPENASSETIO_FWD_DECLARE(ManagerStateBase)
+OPENASSETIO_FWD_DECLARE(managerApi, ManagerInterface)
+OPENASSETIO_FWD_DECLARE(hostApi, HostInterface)
+
+/**
+ * Declare a `RetainPyArgs` alias with common template arguments.
+ *
+ * The template arguments are those types that we expect to be derived
+ * from in Python, and where the associated Python instance(s) must not
+ * be garbage collected, at least until there are no more references in
+ * C++.
+ *
+ * This alias can then be used as a convenience in `.def(...)` calls
+ * without having to re-specify the list of types to match against
+ * the arguments of the decorated function.
+ *
+ * @see PyRetainingSharedPtr
+ * @see RetainPyArgs
+ */
+using RetainCommonPyArgs = openassetio::RetainPyArgs<openassetio::ManagerStateBasePtr,
```

Comment:
This is a great idea to avoid 🦶 🔫  ! nice one.

[consider] Is it worth putting `LoggerInterface` in there too (I think it is just as common as the other Interface classes)?

Reply:
This is added in #522 when it's needed.  Do you think we should pre-empt that and just add it into this PR first?

Reply:
Ahh sorry had missed that, no that's fine. 👍